pkg/constants/constants.go

```diff
@@ -149,3 +149,14 @@ func GetGrubFonts() []string {
 func GetGrubModules() []string {
 	return []string{"loopback.mod", "squash4.mod", "xzio.mod", "gzio.mod", "regexp.mod"}
 }
+
+func GetConfigScanDirs() []string {
```

Comment:
extracted this from main into constants as it makes more sense here. Would probably make more sense to move it to sdk so all get that same dirs..